docs/content/latest/yugabyte-cloud/cloud-faq.md

```diff
@@ -190,6 +192,28 @@ Applications
 
 For more details, refer to [Connect to clusters](../cloud-connect).
 
+### Why is my cluster paused?
```

Comment:
@ddhodge do we need to add on the cluster creation page? Can you check other products (Mongo/Datastax)?

Reply:
will address in a separate PR